File: modelo/paciente.php
<?php
    class Paciente{
        private $id;
        private $nome;
        private $dataNascimento;
        private $endereco;
        private $telefone;
        private $cpf;
        private $observacoes;

        public function getId() {
            return $this->id;
        }

        public function getNome() {
            return $this->nome;
        }

        public function getDataNascimento() {
            return $this->dataNascimento;
        }

        public function getEndereco() {
            return $this->endereco;
        }

        public function getTelefone() {
            return $this->telefone;
        }

        public function getCpf() {
            return $this->cpf;
        }

        public function getObservacoes() {
            return $this->observacoes;
        }

        public function setId($id) {
            $this->id = $id;
        }

        public function setNome($nome) {
            $this->nome = $nome;
        }

        public function setDataNascimento($dataNascimento) {
            $this->dataNascimento = $dataNascimento;
        }

        public function setEndereco($endereco) {
            $this->endereco = $endereco;
        }

        public function setTelefone($telefone) {
            $this->telefone = $telefone;
        }

        public function setCpf($cpf) {
            $this->cpf = $cpf;
        }

        public function setObservacoes($observacoes) {
            $this->observacoes = $observacoes;
        }

        public function __toString(){
            return "Paciente - Id: {$this->id} - Nome: {$this->nome} - Data de nascimento: {$this->dataNascimento} - Endereço: {$this->endereco} - Telefone: {$this->telefone} - CPF: {$this->cpf} - Observações: {$this->observacoes}";
        }
    }
?>

File: src/paciente/Paciente.java
package paciente;

import java.sql.Date;

public class Paciente {
    private int id;
    private String nome;
    private Date dataNascimento;
    private String endereco;
    private String telefone;
    private String cpf;
    private String obs;


    public int getId() {
        return id;
    }
    public void setId(int id) {
        this.id = id;
    }
    public String getNome() {
        return nome;
    }
    public void setNome(String nome) {
        this.nome = nome;
    }
    public Date getDataNascimento() {
        return dataNascimento;
    }
    public void setDataNascimento(Date dataNascimento) {
        this.dataNascimento = dataNascimento;
    }
    public String getEndereco() {
        return endereco;
    }
    public void setEndereco(String endereco) {
        this.endereco = endereco;
    }
    public String getTelefone() {
        return telefone;
    }
    public void setTelefone(String telefone) {
        this.telefone = telefone;
    }
    public String getCpf() {
        return cpf;
    }
    public void setCpf(String cpf) {
        this.cpf = cpf;
    }
    public String getObs() {
        return obs;
    }
    public void setObs(String obs) {
        this.obs = obs;
    }

    public String insert() {
        return "INSERT INTO paciente (nome, data_nascimento, endereco, telefone, CPF, observacoes) VALUES ('" + nome + "', '" + dataNascimento + "', '" + endereco + "', '" + telefone + "', '" + cpf + "', '" + obs + "');";
    }
}
